cmd/generate/config/rules/generic.go

```diff
@@ -19,11 +19,10 @@ func GenericCredential() *config.Rule {
 			"credential",
 			"creds",
 			"key",
-			"passwd",
-			"password",
+			"passw(or)?d",
 			"secret",
 			"token",
-		}, `[\w.=-]{10,150}`, true),
+		}, `(?:[0-9a-z\-_.=]{10,150}|[a-z0-9][a-z0-9+/]{11,}={0,3})`, true),
```

Comment:
```suggestion
		}, `[\w.=-]{10,150}|[a-z0-9][a-z0-9+/]{11,}={0,3}`, true),
```

Since this gets wrapped in a capture group I don't think we need `(?:` and I think we should use `\w` for consistency.

Reply:
I was just referring to the non b64 pattern: `[0-9a-z\-_.=]{10,150}`

Reply:
Oh, duh. Done!